src/core/navigation/Navigation.tsx

```diff
@@ -33,17 +40,15 @@ export type NavigationProps = {
     allLanguages: Language[],
   ) => string;
   getIsItemActive?: (menuItem: MenuItem) => boolean;
-  /** @deprecated Not used anymore i.e. does nothing after HDS 3 was taken into use. */
-  variant?: 'default' | 'inline';
 };
 
-const LOGO_ARIA_LABELS = {
+const CITY_OF_HELSINKI_TRANSLATIONS = {
   EN: 'City of Helsinki',
   FI: 'Helsingin kaupunki',
   SV: 'Helsingfors stad',
 } as const satisfies Record<LanguageCodeEnum, string>;
 
-const LOGO_LABELS = {
```

Comment:
we can add those though hcrc config, in `copy` section we have translations

Reply:
Moved to Config['translations']['helsinkiLogo']